test/base_test_fixture.h

```diff
@@ -1502,6 +1502,38 @@ struct base_test_fixture
         results.get_ref(0, ref);
         REQUIRE(ref == name);
     }
+    void string_vector_test()
```

Comment:
@mloskot it may be worth adding a section to the README.md describing the process of adding to the test suite. You're outline in this comment would be an excellent addition I think!